libs/domains/services/feature/src/lib/hooks/use-helm-service/use-helm-service.ts

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from '@tanstack/react-query'
+import { type Helm } from '@qovery/domains/services/data-access'
+import { queries } from '@qovery/state/util-queries'
+
+export interface UseHelmServiceProps {
+  serviceId?: string
+}
+
+export function useHelmService({ serviceId }: UseHelmServiceProps) {
```

Comment:
we already have a `useService` hook, calling it with `'HELM'` `serviceType` should give you the same result

Reply:
Yes I did that first time and after I created this hook to more clear but it's not really usefull

Reply:
I remember why it was to avoid casting with an `as Helm` each time I used `useService` that's why I created a custom hook for Helm

Reply:
This hook is used once, I'm really not in favor of that. `as` must be avoid as possible, it's literally bypassing the type system.
You can do something like
```ts
  const { data: service } = useService({ serviceId: applicationId, serviceType: 'HELM' })
  
  if (service?.serviceType !== 'HELM') {
    return null
  }
  
  // Here type Helm is inferred
```

Ideally `queries.services.details` should handle `serviceType` with generics and return corresponding service type based on the generic but this is not currently possible with `query-key-factory`